index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>first project</title>
+</head>
```

Comment:
- [ ] The stylesheet does not seem to be linked, which makes the color not reflect. Kindly link the stylesheet to the HTML file 
Hint: `<link rel="" href="">`

![image](https://user-images.githubusercontent.com/43995830/154114834-b189c358-b6cf-4bb0-863e-4cd3e130d3b1.png)